### DISCUSSION TITLE
在文献条目下创建的笔记中插入双链笔记到主笔记时，在文献笔记下只显示一级标题[Feature]

### DISCUSSION THREAD

**Discussion by chenzj36** [windingwind/zotero-better-notes#828] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS:  Windows 11 22H2
- Zotero Version: 6.0.30
- Plugin Version:1.0.4

### Describe the feature request

在文献条目下创建的笔记中插入双链笔记到主笔记时，在文献笔记下只显示一级标题：如“Referred in 主笔记”。而我期望的效果是“Referred in 主笔记/1.2二级标题”。请问可以在哪里设置这个引用的显示格式呢？

**Comment by windingwind** [comment, marked as answer]
https://github.com/windingwind/zotero-better-notes/blob/master/docs/about-note-template.md#builtin-templates

QuickBackLink
